docs/source/ProgrammingGuide/View.rst

```diff
@@ -0,0 +1,723 @@
+.. role:: cppcode(code)
```

Comment:
I appears like I've deleted the file and re-created it, but I think it's because there are so many changes that `git` thinks that's what I've done.

Reply:
Outdated.